src/ua.c

```diff
@@ -693,6 +694,13 @@ void sipsess_conn_handler(const struct sip_msg *msg, void *arg)
 	      sip_transp_name(msg->tp),
 	      &msg->src, &msg->dst);
 
+	if (pl_strncmp(&msg->via.branch, magic_branch, strlen(magic_branch))
```

Comment:
Size can be calculated at compile time in this case:
```suggestion
	if (pl_strncmp(&msg->via.branch, magic_branch, sizeof(magic_branch)-1)
```

Reply:
Thank you! Done.